.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+#Health Connect
+/health-connect/ @abozowski @boutlaw @jessi3py
+
+#Health Platform
+/health-platform-v1/ @abozowski @boutlaw @jessi3py
+
+#Health Services on Wear OS
+/health-services/ @breanatate @yschimke
```

Comment:
I'm not sure I should be a CODEOWNER here.

I'm mainly helping out occasionally.